CMakeLists.txt

```diff
@@ -92,6 +92,12 @@ function(configure_cesium_library targetName)
         CXX_EXTENSIONS NO
     )
 
+    target_compile_definitions( ${targetName} PRIVATE
+        GLM_FORCE_XYZW_ONLY # Disable .rgba and .stpq to make it easier to view values from debugger
+        GLM_FORCE_EXPLICIT_CTOR # Disallow implicit conversions between dvec3 <-> dvec4, dvec3 <-> fvec3, etc
+        GLM_FORCE_SIZE_T_LENGTH # Make vec.length() and vec[idx] use size_t instead of int
```

Comment:
I don't think cesium-native uses `.length()` anywhere, so I'm not too fussed about this. But I am curious: what problem does this one solve for you?

Reply:
`.length` doesn't matter for me that much, but bracket operator does. It's to do `vec[idx]` instead of `vec[int(idx)]` for size_t types.

Reply:
Since cesium-native's use of GLM carries over to libraries that use cesium-native, I'm ok with keeping `int` as the default to minimize disruption. But it is annoying to cast every time when accessing with brackets

Reply:
I really don't have strong feelings on this either way, I was just curious what the motivation was. That makes sense if you frequently have size_t indices.